doc/base/classes.xml

```diff
@@ -5166,7 +5166,7 @@
 			<return type="Array">
 			</return>
 			<description>
-				Return a list of the areas that are totally or partially inside this area.
+				Return a list of the areas ([Area2D]) that are totally or partially inside this area.
```

Comment:
Suggestion: `Returns a list of all the Area2Ds that intersect with this area`

Reply:
Done.